src/base_classes/sport_configs.py

```diff
@@ -0,0 +1,195 @@
+"""
+Sport-Specific Configuration System
+
+This module provides sport-specific configurations including update cadences,
+season characteristics, and sport-specific fields for different sports.
+"""
+
+from typing import Dict, Any, List
+import logging
+
+class SportConfig:
```

Comment:
IDK if this needs to be a dedicated class. This could live within the `Sports` base class and then each config live in the individual Sport class.